Pipfile

```diff
@@ -0,0 +1,50 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[requires]
+python_version = "3.5"
+
+[dev-packages]
+coveralls = ">=1.3.0"
+codecov = ">=2.0.15"
+pytest-cov = "==2.6.0"
+pytest-isort = "==0.2.1"
+pytest-mock = "==1.10.0"
+pytest-pylint = "==0.13.0"
+pytest-runner = "==4.2"
+pytest-dependency = "==0.4.0"
+yapf = "==0.24.0"
+docutils = "==0.14"
+crossbar = "==18.12.1"
+idna = "==2.5"
+onewire = "==0.2"
+pysnmp = "==4.4.6"
+pysnmp-mibs = "==0.1.6"
+xenavalkyrie = "==1.1.2"
+graphviz = "==0.8.4"
+Sphinx = "==1.8.1"
+sphinx_rtd_theme = "==0.4.1"
+pyModbusTCP = "==0.1.7"
```

Comment:
crossbar, idna, onewire, pysnmp(-mibs), xenavalkyrie and pyModbusTCP are normal dependencies.

Reply:
Ok. I will change this.